tests/Kron.py

```diff
@@ -259,7 +356,7 @@ def measureKronInPython(self, objImg, xcen, ycen, nsigma, kfac, nIterForRadius,
 
         return R_K, sumI, 0, False
 
-    def testEllipticalGaussian(self):
+    def xtestEllipticalGaussian(self):
```

Comment:
The way to skip a test is to use the `@unittest.skip("skipping because…")` decorator.